tests/test_octodns_record.py

```diff
@@ -718,6 +718,24 @@ def test_base(self):
         self.assertEquals('www.unit.tests.', ctx.exception.fqdn)
         self.assertEquals(['invalid ttl'], ctx.exception.reasons)
 
+        # value, empty
+        with self.assertRaises(ValidationError) as ctx:
+            Record.new(self.zone, 'www', {
+                'type': 'CNAME',
+                'ttl': 600,
+                'value': ''
+            })
+        self.assertEquals(['missing value'], ctx.exception.reasons)
```

Comment:
Probably want to go ahead and add a `value: None` test here too just to cover that case. It shouldn't happen with the built-in providers, but it'd be possible to get there with someone's internal/custom built provider.